src/components/workspace-utils.js

```diff
@@ -32,11 +32,12 @@ export const useWorkspaces = () => {
   const signal = useCancellation()
   const [loading, setLoading] = useState(false)
   const workspaces = useStore(workspacesStore)
+  const ajax = useContext(ajaxContext)
```

Comment:
It looks like `ajaxContext.Provider` only wraps the Environments component. Will this `useWorkspaces` hook still work where it's used elsewhere in Terra? Maybe wrap `ajaxContext.Provider` around everything in Main.js so the context is available throughout Terra?

Reply:
You are right. Wow, I could have tested at least one other page.